assets/react/v3/entries/course-builder/components/curriculum/TopicHeader.tsx

```diff
@@ -102,11 +101,12 @@ const TopicHeader = ({
     });
 
     if (response.data) {
-      if (response.status_code === 201) {
-        onEdit?.(`topic-${response.data}`);
-      }
       setIsEdit(false);
     }
+
+    if (response.status_code === 201) {
+      onEdit?.(`topic-${response.data}`);
```

Comment:
What if `data` is missing!

Reply:
The response status is
`{
  data: T;
  message: string;
  status_code: number;
}`. One is not dependent on the other.